tendrl/node_agent/discovery/sds/plugins/discover_gluster_sds_details.py

```diff
@@ -43,8 +43,23 @@ def discover_storage_system(self):
         ret_val['detected_cluster_id'] = cluster_id
         ret_val['detected_cluster_name'] = "gluster-%s" % cluster_id
 
+        gluster_get_ver_cmd = NS.config.data.get(
+            "gluster_get_ver_command",
```

Comment:
config?

NS.compiled_definitions.get_parsed_defs()['namespace.tendrl'].get("gluster_get_ver_command")

Reply:
Ah my bad it should be from definitions only. Anyway its not verified yet and discussed this might not be a workable solution and we need more discussion based on BZ# 1559507.